elasticdl/pkg/ps/model.go

```diff
@@ -71,3 +72,25 @@ func (model *Model) InitFromModelPB(pb *proto.Model) error {
 	}
 	return nil
 }
+
+// SaveToModelPB saves in memory model to PB
```

Comment:
`saves in memory model` -> `saves in-memory model`

Reply:
Done